CONTRIBUTING.md

```diff
@@ -86,6 +87,51 @@ assessment.
 
 ## Discussing Mozilla's Position on a Web Specification
 
+### For Mozilla's subject-matter experts
+
+You might be asked for your opinion
+on specifications where you are one of the experts within the Mozilla community,
+or feel like you have valuable contributions to make
+to issues on other specifications even where you haven't explicitly been asked to do so.
+This is the purpose of this repository:
+to help the relevant people in Mozilla's technical leadership come to a conclusion.
+
+The goal of the discussions in this repository is
+to figure out what we think about a specification.
+The key factors to consider are:
+
+* how useful we think the specification is likely to be (or how important the use case it's trying to address is), relative to the likely cost of implementing it and maintaining that implementation
+* whether there are any critical problems we see in the current proposal.
+
+It's useful to discuss these in the relevant issue in this repository.
+
+At the same time,
+we should avoid doing things that move
+discussions that should happen in the standards body's normal discussion mechanisms
```

Comment:
"that move discussions that" sounds a bit odd to me... could that be rephrased in some other way?

Reply:
I at least found "to move discussions that", but didn't see anything better.